docs/content/troubleshooting/csi_ug_troubleshooting_node_crash.md

```diff
@@ -47,30 +47,30 @@ Follow the following procedure to recover from a crashed node (see a [full examp
 
 5.  Verify that the pod is now in a _Running_ state and that the pod has moved to worker-node1.
 
-<a name="full_example">For example:</a>
+    <a name="full_example">For example:</a>
 
-```screen
-$> kubectl get nodes
-NAME STATUS ROLES AGE VERSION
-k8s-master Ready master 6d <your k8s version>
-k8s-node1 Ready <none> 6d <your k8s version>
-k8s-node3 NotReady <none> 6d <your k8s version>
+    ```
+    $> kubectl get nodes
+    NAME STATUS ROLES AGE VERSION
+    k8s-master Ready master 6d <your k8s version>
+    k8s-node1 Ready <none> 6d <your k8s version>
+    k8s-node3 NotReady <none> 6d <your k8s version>
```

Comment:
the column spacing here appears unaligned, in contrast to the [previous one](https://github.com/IBM/ibm-block-csi-driver/blob/8d18ae5a6d95b15894c814e59a27c133aea06d8a/docs/content/troubleshooting/csi_ug_troubleshooting_node_crash.md#identifying-a-crashed-node)

Reply:
fixed. it's not always easy to maintain spacing when moving around various formatting stuff.
DONE